src/FeatureProbe.ts

```diff
@@ -39,6 +41,7 @@ export class FeatureProbe {
   private readonly _repository: Repository;
 
   private readonly _logger: pino.Logger;
+  private _socket?: Socket<any, any>;
```

Comment:
use `DefaultEventsMap` to replace `any`

import { DefaultEventsMap } from "@socket.io/component-emitter";

private _socket?: Socket<DefaultEventsMap, DefaultEventsMap>;